src/CMakeLists.txt

```diff
@@ -0,0 +1,165 @@
+set(LIBRARY_NAME "nuklear")
+
+include(GNUInstallDirs)
+
+set(pub "nuklear.h")
+source_group("Public Header Files" FILES "${pub}")
+
+set(priv1
+        "nuklear_internal.h" "nuklear_math.c" "nuklear_util.c"
+        "nuklear_color.c" "nuklear_utf8.c" "nuklear_buffer.c"
+        "nuklear_string.c" "nuklear_draw.c" "nuklear_vertex.c")
+source_group("Private [1] Files" FILES "${priv1}")
+
+set(extern "stb_rect_pack.h" "stb_truetype.h")
+source_group("External Files" FILES "${extern}")
+
+set(priv2
+        "nuklear_font.c" "nuklear_input.c" "nuklear_style.c"
+        "nuklear_context.c" "nuklear_pool.c" "nuklear_page_element.c"
+        "nuklear_table.c" "nuklear_panel.c" "nuklear_window.c"
+        "nuklear_popup.c" "nuklear_contextual.c" "nuklear_menu.c"
```

Comment:
I understand glob is frowned upon in CMake, but it may help ease the maintenance of the CMake definition files.

Reply:
The reason why glob is not recommended is because it is slower, and when there is a file added or removed you do not trigger a cmake rebuild.

Instead of using glob, I would suggest to format these lists differently (one file per line) so that editing the list results in cleaner diffs.